static/js/components/ProgressWidget.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import CircularProgressWidget from './progress/CircularProgressWidget';
+
+export default class ProgressWidget extends React.Component {
+
+  static propTypes = {
+    actual:       React.PropTypes.number,
+    total:        React.PropTypes.number,
+    disabled:     React.PropTypes.bool
+  };
```

Comment:
Can you use flow types instead of react `propTypes` for typing the props on the component?